### PR TITLE
Feature/fetch flow execution

### LINKED ISSUE #21: Add Fetch a Flow Execution API cal
Add the corresponding API call, as [documented in the Azkaban project](https://azkaban.github.io/azkaban/docs/latest/#api-fetch-a-flow-execution)

`Given an execution id, this API call fetches all the detailed information of that execution, including a list of all the job executions.`

### PR DESCRIPTION
**Pull request by aguimaraesduarte**
Closes: #21 
Adds Fetch a Flow Execution API call with parsing of return

### CODE REVIEW

File: azkaban_cli/azkaban.py [hunk 1]
```diff
@@ -540,3 +541,31 @@ def __check_group_permissions(self, permission_options):
...
+
+    def fetch_flow_execution(self, execution_id):
+        """Fetch a flow execution command, intended to make the request to Azkaban
+        and treat the response properly.
+
+        This method receives the execution id, makes the fetch a flow execution request
+        to fetch the flow execution details and evaluates the response.
+
+        If execution_id is wrong or if there is no session_id, it returns false.
+        If everything is fine, returns True.
```
Comment (ggalves) [inline azkaban_cli/azkaban.py]: This doesn't look correct

File: azkaban_cli/azkaban_cli.py [hunk 2]
```diff
@@ -271,6 +272,50 @@ def __change_permission(ctx, project, group, admin, read, write, _execute, _sche
     except ChangePermissionError as e:
         logging.error(str(e))
 
+def __parse_flow_execution(json):
```
Comment (ggalves) [inline azkaban_cli/azkaban_cli.py]: maybe just a matter of opinion, but i would prefer something like __log_flow_execution, I think it is more suggestive

File: azkaban_cli/azkaban.py (hunk 1 above)
Comment (aguimaraesduarte) [reply azkaban_cli/azkaban.py]: You're right. I fixed the docstring now.

File: azkaban_cli/azkaban_cli.py (hunk 2 above)
Comment (aguimaraesduarte) [reply azkaban_cli/azkaban_cli.py]: Fine by me. Fixed in new commit.

**Comment by ggalves** [comment]
@aguimaraesduarte LGTM!

I've merged your last PR and now this PR has conflicts. Can you solve it?

**Comment by aguimaraesduarte** [comment]
@ggalves I fixed the conflicts with the new push

**Pull request merged**